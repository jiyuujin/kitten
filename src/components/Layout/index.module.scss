body {
  font-family: Futura, 'Trebuchet MS', Arial, sans-serif;
  color: #000;
  background-color: #fff;
}

@media (prefers-color-scheme: dark) {
  body {
    color: #fff;
    background-color: #303030;
  }
}
