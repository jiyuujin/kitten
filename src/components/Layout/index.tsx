import React from 'react'

export default ({ children }: { children?: React.ReactNode }) => (
    <main>{children}</main>
)
