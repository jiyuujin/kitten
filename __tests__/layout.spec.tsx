import React from 'react'
import renderer from 'react-test-renderer'
import Layout from '../src/components/Layout'

const children: React.ReactChild = ''

describe('Layout', () => {
  it('render correctly', () => {
    const tree = renderer.create(<Layout>{children}</Layout>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
