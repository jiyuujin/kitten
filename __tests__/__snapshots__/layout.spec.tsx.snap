// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout render correctly 1`] = `
<div>
  
</div>
`;
